src/Handlers/CollectHandlers.php

```diff
@@ -52,6 +52,20 @@ public function middleware($callable): void
         array_unshift($this->globalMiddlewares, $callable);
     }
 
+    /**
+     * @return $this
+     */
+    public function reset(): static
+    {
+        $this->handlers = [];
+        $this->groups = [];
+        $this->globalMiddlewares = [];
+        $this->groupHandlers = [];
+        $this->target = 'handlers';
+
+        return $this;
+    }
+
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏this violates the principle that a nutgram instance is immutable. if something change the entire instance must be scrapped and rebuilded. Does not take into account the config might change also

Reply:
how can we rebuild the nutgram instance?

Reply:
https://github.com/nutgram/nutgram/pull/714/files
https://github.com/nutgram/laravel/pull/24/files

Those is what you were trying to do?

Reply:
Yes, thanks. but i think this will break `nutgram:run` command.